src/test/java/io/kokuwa/maven/helm/junit/MojoExtension.java

```diff
@@ -106,12 +108,20 @@ public Object resolveParameter(ParameterContext parameterContext, ExtensionConte
 			// preconfigure
 
 			mojo.setChartDirectory(new File("src/test/resources/simple")); // set some sane defaults for tests
-			mojo.setHelmExecutableDirectory(new File("src/it")); // avoid download helm
+			mojo.setHelmExecutableDirectory(determineHelmExecutableDirectory());
 			mojo.setHelmVersion("3.12.0"); // avoid github api
 
 			return mojo;
 		} catch (ReflectiveOperationException e) {
 			throw new ParameterResolutionException("Failed to setup mockito.", e);
 		}
 	}
+
+	public static File determineHelmExecutableDirectory() {
+		if (System.getProperty("os.arch").equals("aarch64")) {
+			return new File("src/it/helm-executables/aarch64-helm/");
+		} else {
+			return new File("src/it/helm-executables/x86_64-helm/");
+		}
+	}
```

Comment:
simple logic for testing the machine's architecture, then determining which helm executable to use accordingly

Reply:
Should add javadocs to the method with this info

Reply:
added